resdk/resources/sample.py

```diff
@@ -313,3 +313,22 @@ def is_background(self):
             self._is_background = len(background_relations) > 0  # pylint: disable=len-as-condition
 
         return self._is_background
+
+    def duplicate(self, inherit_collections=False):
+        """Duplicate (make copy of) ``sample`` object.
+
+        :param inherit_collections: If ``True`` then duplicated
```

Comment:
Added `inherit_collections` parameter.

Reply:
This was renamed to `inherit_collection` in latest resolwe.